### ISSUE TITLE
synthesize new background problem

### ISSUE THREAD

**Issue by imerse** [PeterL1n/BackgroundMattingV2#60]
Excellent work! Excuse me, if I use a single background for synthesis, there is no problem, and the matting is normal, but if I use a complex background for synthesis, there will be such problematic edges. What should I do in this situation? Has anyone encountered it?

After I read the background image, I used the following formula to synthesize:
`com = pha * fgr + (1 - pha) * torch.tensor([brgcom], device='cuda')`

![image](https://user-images.githubusercontent.com/8415005/107339614-d6508680-6af7-11eb-84dd-91a1198025fd.png)
![image](https://user-images.githubusercontent.com/8415005/107339671-e8322980-6af7-11eb-9690-9659f53fb92f.png)
![image](https://user-images.githubusercontent.com/8415005/107339745-013ada80-6af8-11eb-86a0-8aba350c8f37.png)

**Comment by PeterL1n** [comment]
Can you label the three images as what they are? If the first one is the produced alpha, then the alpha does not look like having any problem. So the problem is not the model, but your compositing script?

**Comment by imerse** [comment]
Sorry, yes, there is no problem with the model, it is a synthesis problem, I solved this problem. Thanks！
